entrypoint.sh

```diff
@@ -51,6 +51,19 @@ case ${RUNNER_SCOPE} in
     ;;
 esac
 
+# Loading the files from the mounted directory
+if [ -d ${CONFIGURED_ACTIONS_RUNNER_FILES_DIR} ]; then
+  cp -p -r ${CONFIGURED_ACTIONS_RUNNER_FILES_DIR}/. "/actions-runner"
+fi
+
+if [ -f "/actions-runner/.runner" ]; then
```

Comment:
this is new functionality, id rather copy the contents and behave as before, not change behavior from previous

running `$@` is already called at the last part of entrypoint.sh, and exit 0 is implicit already

Reply:
I do not really get your suggestion but placing `$@; exit 0` is meant as a return statement for avoiding the runner reconfiguration

Reply:
I mean we alreaady have a $@ and an exit 0

We should not have multiple run/exits but let it trickle through to the end

Reply:
Then only the way to put all the configuration code under `if` statement

Reply:
Why are you ignoring SC2068 instead of `"$@"`?

Reply:
I dont remember why SC2068, i think there was an interpolation error with docker run with that

Reply:
This warning is about the following difference:

## Case 1:

You have `script.sh` with the following content:
```
exec $@
```

Then `sh script.sh "a" "b c"` cause the execution of the following command: `exec a b c`

## Case 2:

You have `script.sh` with the following content:
```
exec "$@"
```

Then `sh script.sh "a b" "c"` cause the execution of the following command: `exec "a" "b c"`

So SC2068 warn that you are falling into the first case. not the second

Reply:
So `$@` can be changed to `"$@` and there is no need to ignore SC2068